Add `osu!mania 4K World Cup 2024: Quarterfinals Recap` news post (#12125)
ppy/osu-wiki: news/2024/2024-09-11-osumania-4k-world-cup-2024-quarterfinals-recap.md

---
layout: post
title: "osu!mania 4K World Cup 2024: Quarterfinals Recap"
date: 2024-09-11 21:00:00 +0000
---

> Honey, it's time to consume your weekly dose of World Cup recaps...

Reviewer: i don't think this tone is quite appropriate

Author: c32debfb52f88cf540adabaa99a5176055202f15 changed to something more appropriate

![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)

> We are slowly reaching the end stages of MWC 4K, which means the map pools are getting harder, and the top teams are getting chances to fully flex their skills to both their opponents, and the community at large. If you have not kept up with this tournament so far, now is the ideal time to get started.

Reviewer: like elsewhere

```suggestion
We are slowly reaching the end stages of MWC 4K, which means the mappools are getting harder, and the top teams are getting chances to fully flex their skills to both their opponents, and the community at large. If you have not kept up with this tournament so far, now is the ideal time to get started.
```

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

[**Download the pool here! (110 MB)**](https://osu.ppy.sh/beatmaps/packs/P259)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2240120291&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> This time around we're not seeing any new slots put into the mix, but that's not to say the pool hasn't stepped up in difficulty by a surprising amount! Most notably, [sasakure.UK x TJ.hangneil - raputa](https://osu.ppy.sh/beatmapsets/2242647#mania/4766767) by [[GB]V1do](https://osu.ppy.sh/users/17527968) (HB1) bumps up to 242BPM, which begins to show major weaknesses for many players and their physicality, alongside [The Flashbulb - A Raw Understanding](https://osu.ppy.sh/beatmapsets/1632098#mania/4766799) by [Pizza69](https://osu.ppy.sh/users/25285704) (RC4) really signifying the challenge behind purely technical patterning, which is no joke when uprated to 1.15x!

Reviewer: ```suggestion
This time around we're not seeing any new slots put into the mix, but that's not to say the pool hasn't stepped up in difficulty by a surprising amount! Most notably, [sasakure.UK x TJ.hangneil - raputa](https://osu.ppy.sh/beatmapsets/2242647#mania/4766767) by [[GB]V1do](https://osu.ppy.sh/users/17527968) (HB1) bumps up to 242 BPM, which begins to show major weaknesses for many players and their physicality, alongside [The Flashbulb - A Raw Understanding](https://osu.ppy.sh/beatmapsets/1632098#mania/4766799) by [Pizza69](https://osu.ppy.sh/users/25285704) (RC4) really signifying the challenge behind purely technical patterning, which is no joke when uprated to 1.15x!
```

To round up the mappool, we've also got one new [osu! original](/wiki/osu!_originals)!

[![](/wiki/shared/news/2024-09-11-osumania-4k-world-cup-2024-quarterfinals-recap.md/tree_of_misery.jpg)](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/359/Sorrow%20Mania/cygnus%20-%20Tree%20of%20Misery.mp3">
    </audio>
</div>

Many players across all game modes are familiar with the player, mapper, and artist [cygnus](https://osu.ppy.sh/beatmaps/artists/359), who goes by [Matrix](https://osu.ppy.sh/users/5052899) on osu!, and this time around they're debuting in MWC 4K with [Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) charted by [Miaurichesu](https://osu.ppy.sh/users/10055648)!

> With it taking up the chordjack slot (RC3) for the round, you can be sure to expect it to ask nothing more than for you to simply be accurate due to it being arguably the most accuracy intensive pick out of all pools, thanks to the straightforward nature behind them. You definitely shouldn't underestimate the contents of the chart however, as the chart has ZERO breaks, and it only gets heavier in patterning the further you go. This, along with added stamina management makes it so the slightest dent in your score can cause a chain reaction of misses or other judgements that would take away a large lead in score.

Reviewer: ```suggestion
Taking up the chordjack slot for the round (RC3), it's expecting nothing more from players than straightforward accuracy. You definitely shouldn't underestimate the contents of the chart however, as the chart has *zero* breaks, and it only gets heavier in patterning the further you go. The slightest dent in your score can cause a chain reaction of misses or tank your accuracy if you don't manage your stamina!
```

| **Brazil** ::{ flag=BR }:: | **6** | 5 | ::{ flag=MX }:: Mexico |
| Philippines ::{ flag=PH }:: | 2 | **6** | ::{ flag=CL }:: **Chile** |
| **United States** ::{ flag=US }:: | **6** | 0 | ::{ flag=MY }:: Malaysia |
| **Taiwan** ::{ flag=TW }:: | **6** | 2 | ::{ flag=NZ }:: New Zealand |
| **South Korea** ::{ flag=KR }:: | **6** | 1 | ::{ flag=VN }:: Vietnam |
| **Russian Federation** ::{ flag=RU }:: | **6** | 2 | ::{ flag=TW }:: Taiwan |
| **Thailand** ::{ flag=TH }:: | **6** | 1 | ::{ flag=SG }:: Singapore |
| Brazil ::{ flag=BR }:: | 0 | **6** | ::{ flag=JP }:: **Japan** |
| United Kingdom ::{ flag=GB }:: | 5 | **6** | ::{ flag=HK }:: **Hong Kong** |
| **China** ::{ flag=CN }:: | **6** | 1 | ::{ flag=CA }:: Canada |

## Match highlights

### ::{ flag=BR }:: Brazil vs. ::{ flag=MX }:: Mexico

[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115406325)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2245307636&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Brazil and Mexico is a match-up viewers were excited for. Both teams are generally all-rounders, meaning that this match was anyone's game, especially with each team's seeding being #12 and #18 respectively. If you are familiar with the Brazil team from previous years, you may know that they are former winners of the Mania World Cup back in 2020 and 2021. Mexico on the other hand are more or less newcomers who made their first appearance in the main brackets in the 2023 rendition of the world cup, so if Mexico wanted to prove that they could take down any fight, this was the match that they had to win.

Reviewer: ```suggestion
Brazil and Mexico is a matchup viewers were excited for, as it could've been anyone's game for the closely seeded teams, both featuring all-rounders. Regular spectators may recall Brazil's past victories in the osu!mania 4K World Cup back in 2020 and 2021.

Team Mexico on the other hand is composed more or less of newcomers who made their first appearance in the main brackets in MWC 4K 2023, so if Mexico wanted to prove that they could take down any fight, this was the match that they had to win.
```

> - [DJ SHARPNEL - KLASSIKER EINHEIT (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2242540#mania/4766576) ([YuEast 2018](https://osu.ppy.sh/users/13953619), LN2) @ [30:36](https://www.twitch.tv/videos/2245230506?t=00h30m36s) - The score was 1–2 with no breakpoints in sight. Brazil ended up picking LN2, which contains very dense LN sections with no breaks, so stamina is needed if you want a strong performance on this beast of a map. Unfortunately for Brazil, they had a couple of slip ups at the end of the map, letting Mexico barely get the point by 2,930 score, resulting in the first breakpoint of the match.

Reviewer: ```suggestion
- [DJ SHARPNEL - KLASSIKER EINHEIT (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2242540#mania/4766576) ([YuEast 2018](https://osu.ppy.sh/users/13953619), LN2) @ [30:36](https://www.twitch.tv/videos/2245230506?t=00h30m36s) - The score was 1–2. Brazil ended up picking the densely mapped LN2, requiring strong stamina for a good performance. Unfortunately for Brazil, a couple slip-ups at the end of the map let Mexico barely get the first break point by 2,930 score.
```

> - [cygnus - Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) ([Miaurichesu](https://osu.ppy.sh/users/10055648), RC3) @ [48:56](https://www.twitch.tv/videos/2245230506?t=00h48m56s) - With the score being 2–4, Brazil needed to step up their game and take a breakpoint from Mexico if they wanted any chance of winning the match. Mexico ended up choosing RC3, a map riddled with consistent chordjacks throughout the map, with consistency being key. This was Brazil's only chance at winning the match, or else it could end in a disaster for them. Thankfully for Brazil, fortune was on their side this time around, managing to take the breakpoint from Mexico while not breaking a sweat.

Reviewer: ```suggestion
- [cygnus - Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) ([Miaurichesu](https://osu.ppy.sh/users/10055648), RC3) @ [48:56](https://www.twitch.tv/videos/2245230506?t=00h48m56s) - Mexico, leading with 2–4, ended up choosing RC3, a consistency chart riddled with chordjacks. This was Brazil's only chance at stealing a break point and turning the match around, and thankfully for Brazil, fortune was on their side.
```

> - [Kou! - Frost Walker](https://osu.ppy.sh/beatmapsets/2242668#mania/4766814) ([guden](https://osu.ppy.sh/users/11626065), TB) @ [1:13:44](https://www.twitch.tv/videos/2245230506?t=01h13m44s) - Knock knock, it's tiebreaker time! This map has been brought to you by the Catch World Cup, with its custom Grand Finals tiebreaker song, Frost Walker, from earlier this year. This tiebreaker was way too close throughout the entire map, never going over a score difference of 10k. After almost seven minutes of both teams fighting through the cold atmosphere, Brazil managed to take the win with a score difference of 5,670.

Reviewer: ```suggestion
- [Kou! - Frost Walker](https://osu.ppy.sh/beatmapsets/2242668#mania/4766814) ([guden](https://osu.ppy.sh/users/11626065), TB) @ [1:13:44](https://www.twitch.tv/videos/2245230506?t=01h13m44s) - Knock knock, it's tiebreaker time! This chart has been brought to you by the osu!catch World Cup, with its original Grand Finals tiebreaker song. This match itself was way too close throughout, never going over a score difference of 10k. After almost seven minutes of both teams fighting through the cold atmosphere, Brazil managed to take the win with a score difference of 5,670.
```

> Both teams played amazing this match, with each pick being so close. While Brazil managed to take the win and will be moving on to the Quarterfinals, we can't say the same for Mexico as they are now out of the tournament. We wish Brazil the best of luck going into the Quarterfinals, but for Mexico, we hope they will be able to place better next year.

Reviewer: ```suggestion
Both teams gave it their all this match, with each pick being incredibly close. While Brazil managed to take the win and will be moving on to the Quarterfinals, we can't say the same for Mexico as they are now out of the tournament. We wish Brazil the best of luck and hope Mexico will be able to place better next year.
```

---
layout: post
title: "osu!mania 4K World Cup 2024: Quarterfinals Recap"
date: 2024-09-11 21:00:00 +0000
---

> With the Quarterfinals weekend freshly concluded, now is the ideal time to join us in reviewing the best moments of yesteryear, and prepare for the upcoming Semifinals.

Reviewer: yesteryear?

```suggestion
With the Quarterfinals weekend freshly concluded, now is the ideal time to join us in reviewing the best moments and prepare for the upcoming Semifinals.
```

![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)

> We are slowly reaching the end stages of MWC 4K, which means the mappools are getting harder, and the top teams are getting chances to fully flex their skills to both their opponents, and the community at large. If you have not kept up with this tournament so far, now is the ideal time to get started.

Reviewer: ```suggestion
We are slowly reaching the end stages of MWC 4K, which means the mappools are getting harder, and the top teams are getting chances to fully flex their skills to both their opponents and the community at large. Let's see who are making the headlines.
```

- [Frums - gur yvsr](https://osu.ppy.sh/beatmapsets/2242442#mania/4766318) ([Vortex-](https://osu.ppy.sh/users/4999669), RC6) @ [20:00](https://www.twitch.tv/videos/2245925227?t=00h20m00s) - This was the UK's second pick, but far from a sure one. The polyrhythms and technical pattern barrages in this map are nothing to scoff at, and comboing through them is quite a feat, let alone getting high accuracy! Things came to a very tense finish, with a Hong Kong advantage through the map being wildly swung in the last few seconds, and ending with a narrow British loss by a mere 2400 points.
- [Ushirokara Haiyoritai G - Koi wa Chaos no Shimobenari](https://osu.ppy.sh/beatmapsets/2242646#mania/4766766) ([0DZ0](https://osu.ppy.sh/users/28156707), RC5) @ [30:33](https://www.twitch.tv/videos/2245925227?t=00h30m33s) - This point was a masterclass, and the UK was the teacher. A short and sweet technical map, as seen towards the weekend a single miss here means so much. Hong Kong misses. The UK doesn't. They played this map 2 times in this weekend. Those are the top 2 team scores out of every team in the tournament. Enough said.

> - [Kou! - Frost Walker](https://osu.ppy.sh/beatmapsets/2242668#mania/4766814) ([guden](https://osu.ppy.sh/users/11626065), TB) @ [62:40](https://www.twitch.tv/videos/2245925227?t=01h02m40s) - From a 5–2 lead, and after a tragic breakpoint in the chordjack pick, the UK finds themselves on 5–5. The devastating array of patterns laid out in this collab ensues, and after the dust settles, some of the LN patterns proved to be too much for the Brits, leading to a pretty narrow 7900 points loss. Actually, this is the second tiebreaker each of these teams play in the tournament — and their positions seem clear. 2 wins for Hong Kong. 2 losses for the United Kingdom.

Reviewer: ```suggestion
- [Kou! - Frost Walker](https://osu.ppy.sh/beatmapsets/2242668#mania/4766814) ([guden](https://osu.ppy.sh/users/11626065), TB) @ [62:40](https://www.twitch.tv/videos/2245925227?t=01h02m40s) - From a 5–2 lead, and after a tragic break point in the chordjack pick, the UK found themselves at 5–5. After the dust settled from a devastating fusillade of LN patterns, some of them proved to be too much for the Brits, leading to a pretty narrow 7,900-point loss. Actually, this was the second tiebreaker each of these teams played in the tournament, and their positions seem clear. 2 wins for Hong Kong. 2 losses for the United Kingdom.
```

> Commiserations to the British team for their run this year, unfortunately ended by 2 insanely close matches. Congratulations to Hong Kong for fighting on against all odds, next weekend they will face the Philippines in hopes of keeping it alive once more.

Reviewer: ```suggestion
Commiserations to the British team for their run this year, unfortunately ended by 2 insanely close matches. Congratulations to Hong Kong for fighting on against all odds — next weekend they will face the Philippines in hopes of keeping it alive once more.
```

## Score highlights

There's no denying that all teams thus far have given their all, especially as we draw ever so closer to the lattermost rounds of the tournament. But this round's standout amongst all teams must've been Japan!

> - ::{ flag=JP }:: [[Crz]yomogi237](https://osu.ppy.sh/users/28571440) took everyone by surprise with their crazy ratio in [cygnus - Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) by [Miaurichesu](https://osu.ppy.sh/users/10055648) (RC3), achieving a score of 999,792 with only **10 perfects** in their match [against Spain](https://www.twitch.tv/videos/2244890379).

Reviewer: i guess we're using lazer terms? note for stable it'd be 300 and rainbow 300

```suggestion
  - ::{ flag=JP }:: [[Crz]yomogi237](https://osu.ppy.sh/users/28571440) took everyone by surprise with their crazy ratio in [cygnus - Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) by [Miaurichesu](https://osu.ppy.sh/users/10055648) (RC3), achieving a score of 999,792 with only **10 greats** in their [match against Spain](https://www.twitch.tv/videos/2244890379).
```

> - ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) covered Japan's SV needs with their astonishing 997,856 score on [Tanger - BREAKME](https://osu.ppy.sh/beatmapsets/2242663#mania/4766804) by [Promachos](https://osu.ppy.sh/users/14069486) (SV1) in their match [versus Brazil](https://www.twitch.tv/videos/2245937607).

Reviewer: ```suggestion
- ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) covered Japan's SV needs with their astonishing 997,856 score on [Tanger - BREAKME](https://osu.ppy.sh/beatmapsets/2242663#mania/4766804) by [Promachos](https://osu.ppy.sh/users/14069486) (SV1) in their [match versus Brazil](https://www.twitch.tv/videos/2245937607).
```

Will they be able to hold up next week? Will the other teams keep their edge? Whatever the case may be, Japan has proved that they are a force to be reckoned with.

---

> Tune in next week for the Semifinals recap, or even better, watch the matches live on the [osu!live Twitch channel](https://www.twitch.tv/osulive)! As always, you can browse the [official wiki page](/wiki/Tournaments/MWC/2024_4K), check out the schedule along with the full tournament details, and pick which teams you want to support in their quest to become champions.

Reviewer: i usually want to omit using "official" because in most cases it's not necessary to disambiguate anything and is therefore meaningless

```suggestion
Tune in next week for the Semifinals recap, or even better, watch the matches live on the [osu!live Twitch channel](https://www.twitch.tv/osulive)! As always, you can browse the [wiki page](/wiki/Tournaments/MWC/2024_4K), check out the schedule along with the full tournament details, and pick which teams you want to support in their quest to become champions.
```
